config/schema.yaml

```diff
@@ -1,53 +1,105 @@
 ---
-sample_id:
-  type: 'String'
-  required: true
```

Comment:
**comment (blocking)**: Sample ID seems to have been removed

Reply:
Oops, thanks for catching this. Should be back https://github.com/uclahs-cds/pipeline-StableLift/pull/30/commits/ad7bccdc0f00310c318ade4b3cae5b7eea67d367